counterparty-core/counterpartycore/lib/messages/dispense.py

```diff
@@ -88,8 +93,8 @@ def parse(db, tx):
     cursor = db.cursor()
 
     outs = []
-    if util.enabled("multiple_dispenses"):
-        outs = ledger.get_vouts(db, tx["tx_hash"])
+    if protocol.enabled("multiple_dispenses"):
+        outs = ledger.ledger.get_vouts(db, tx["tx_hash"])
```

Comment:
this is a real nitpick but I don't love `x.x` imports. feels redundant

Reply:
(but I'm totally fine with it!)

Reply:
yeah completely agree! I need to split ledger.py is several files.. will be ledger.balances.* ledger.transactions.* etc..